tests/zkevm/test_worst_compute.py

```diff
@@ -680,6 +682,17 @@ def test_worst_modexp(state_test: StateTestFiller, pre: Alloc, fork: Fork):
             ],
             id="bls12_fp_to_g2",
         ),
+        pytest.param(
```

Comment:
Test is going to pass but the test is only going to test empty account calls because there's no precompile here prior to Prague.

We should test whether `precompile_address` is in `fork.precompiles()`, and if not just `pytest.skip("Precompile not enabled")` at the start of the function.